.github/workflows/pr-dependencies.yml

```diff
@@ -0,0 +1,12 @@
+name: Check PR Dependencies
```

Comment:
This is already an org-wide action (as I had stated in the issue).  It's not needed explicitly here.  Please remove.

Reply:
Okay I will be removing it.